mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/controller/MockThunderheadIAMController.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.thunderhead.controller;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+@RestController
+public class MockThunderheadIAMController {
+
+    @CrossOrigin(origins = "*")
+    @PostMapping("/iam/getUser")
+    public HashMap<String, String> getUser(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
```

Comment:
Not a big deal, perhaps using the interface `Map` for the return type would be better.

Reply:
thx, done